pkg/osm/types.go

```diff
@@ -14,3 +14,6 @@ type TrafficSplitVersion string
 
 // HTTPRouteVersion is a string type alias for the SMI HTTPRoute version supported.
 type HTTPRouteVersion string
+
+// Annotation is a string type alias for the an osm annotation.
```

Comment:
// Annotation is a string type alias for the osm annotation.

Reply:
Good catch!

Reply:
Resolved.